python/pyiceberg/avro/file.py

```diff
@@ -93,49 +96,52 @@ def get_schema(self) -> Schema:
             raise ValueError("No schema found in Avro file headers")
 
 
+F = TypeVar("F", bound=StructProtocol)
```

Comment:
What does `F` represent? I'd expect something like `T` (type) or `R` (record?). Avro itself uses `D` for datum, which may be a good idea.

Reply:
It was `Field`, I wanted to pick a letter that wasn't used regularly. Let's go for D instead, I like it!